### PR TITLE
 Added new features to the description

### PR DESCRIPTION
**Pull request by vantihovich**

### CODE REVIEW

File: swagger/api/apiauth.yaml [hunk 1]
```diff
@@ -83,6 +83,64 @@ paths:
...
+
+  /auth/deactivate:
+    post:
+      summary: Deactivate user
+      description:  Deactivation for user not to be able to login
+      operationId: deactivateUser
+      tags:
+        - users
+
+      requestBody:
```
Comment (apaliavy) [inline swagger/api/apiauth.yaml:99]: Does it mean that any user in the system will be able to deactivate any another user? 🤔

File: swagger/api/apiauth.yaml [hunk 2]
```diff
@@ -83,6 +83,64 @@ paths:
...
+        "200":
+          description: deactivated cuccessfully 
+        "400":
+          description: Invalid request body   
+        "403":
+          description: Invalid login - doesn`t exist in DB
+        "500":
+          description: Internal server error
+
+  /auth/passwordReset:
```
Comment (apaliavy) [inline swagger/api/apiauth.yaml:119]: password_reset

File: swagger/api/apiauth.yaml [hunk 3]
```diff
@@ -83,6 +83,64 @@ paths:
...
+      tags:
+        - users
+
+      requestBody:
+        description: login and new password
+        required: true
+        content: 
+          application/json:
+            schema: 
+              $ref: '#/components/schemas/LoginRequest'
```
Comment (apaliavy) [inline swagger/api/apiauth.yaml:133]: Not implemented yet, or you gonna use email+password for password reset?

File: swagger/api/apiauth.yaml (hunk 2 above)
Comment (vantihovich) [reply swagger/api/apiauth.yaml]: done

File: swagger/api/apiauth.yaml [hunk 4]
```diff
@@ -117,8 +289,99 @@ components:
...
+    ForgotPWDRequestPt2:
+      type: object
+      properties:
+        secret: 
+          type: string
+      required:
+          - secret
+      example:
+          secret: blablabla234
+    ForgotPWDResponsePt2:
```
Comment (apaliavy) [inline swagger/api/apiauth.yaml:322]: Bad naming

File: swagger/api/apiauth.yaml [hunk 5]
```diff
@@ -117,8 +289,99 @@ components:
...
+        secret: 
+          type: string
+      required:
+          - user_id
+          - secret
+      example:
+          user_id: user1
+          secret: blablabla234
+
+    ForgotPWDRequestPt3:
```
Comment (apaliavy) [inline swagger/api/apiauth.yaml:336]: Same here

**Pull request merged**